### ISSUE TITLE
frn code？which one？1d? 2d? conver?

### ISSUE THREAD

**Issue by henbucuoshanghai** [tattaka/Filter-Response-Normalization-PyTorch#3]
where is the code?
how to use it?

**Comment by tattaka** [comment]
All the code is [filter_response_normalization](https://github.com/tattaka/Filter-Response-Normalization-PyTorch/tree/master/filter_response_normalization). 
You can use this directory as a module.
The `convert_model` function invalidates the ReLU in the pytorch model and replaces BN according to the dimension.  Please see [README.md](https://github.com/tattaka/Filter-Response-Normalization-PyTorch/blob/master/README.md) for usage.
But I recommend that you manually replace BN and ReLU with FRN and TLU, as the performance of the model may change.

**Comment by henbucuoshanghai** [comment]
1d? 2d?what is the differemt？

**Comment by tattaka** [comment]
The dimensions input to each function are different.
It corresponds to `torch.nn.BatchNormxd`.
